frontend/src/pages/Interventions/SecondActivity.tsx

```diff
@@ -156,61 +181,84 @@ const StyledCheckOutlined = styled(CheckOutlined)`
   margin-top: 20px;
 `;
 
-const correctUrl = {
-  url:
-    "https://firebasestorage.googleapis.com/v0/b/vocab-buddy-53eca.appspot.com/o/jSyyDnxzx41VFQNQbbEw%2Fminiscule3.png?alt=media&token=cb4f4cf6-a1d0-465d-a972-087230d2ff05",
-  correct: true,
-};
+const CloudImageLeft = styled(CloudImage)`
+  position: absolute;
+  left: 0;
+  width: 15%;
```

Comment:
I wonder if we want to make a like component that adds the cloud images so that all of these that need it gets the cloud from that Component, kind of like Layout now.